README.md

```diff
@@ -8,18 +8,25 @@
 ![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
 [![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
 
-SkyPilot is a framework for easily running machine learning[^1] workloads on any cloud through a unified interface. No knowledge of cloud offerings is required or expected – you simply define the workload and its resource requirements, and SkyPilot will automatically execute it on AWS, Google Cloud Platform or Microsoft Azure.
+SkyPilot is a framework for easily running machine learning workloads[^1] on any cloud. 
 
 
-### Key features
+Use the clouds **easily** and with **low cost**, all without needing to learn cloud infra.
+
+
+
+_Ease of use & productivity_
 * **Run existing projects on the cloud** with zero code changes
-* **No cloud lock-in** – seamlessly run your code across different cloud providers (AWS, Azure or GCP)
-* **Minimize costs** by leveraging spot instances and automatically stopping idle clusters
-* **Automatic recovery from spot instance failures**
-* **Automatic fail-over** to find resources across regions and clouds
+* **Easily manage jobs** across multiple clusters
+* **Automatic fail-over** to find scarce resources (GPUs) across regions and clouds
 * **Store datasets on the cloud** and access them like you would on a local file system 
-* **Easily manage job queues** across multiple clusters
+* **No cloud lock-in** – seamlessly run your code across different cloud providers (AWS, Azure or GCP)
 
+_Cost saving_
+* Run jobs on **spot instances** with **automatic recovery** from preemptions
+* Hands-free life cycle: **automatically stop idle clusters**
+* One-click use of **TPUs**, for high-performance, cost-effective training
```

Comment:
Not sure if TPUs are cheaper (may be [more expensive for some models](https://indico.cern.ch/event/1015407/contributions/4261881/attachments/2209662/3739396/20210316_IML_TPUvsGPU.pdf), [cheaper for others](https://eng.snap.com/en-US/training-models-with-tpus)). Maybe move to ease of use heading?

Reply:
I believe this is increasingly the case and propose we keep it.  For example, Google internally all but abandons GPUs, from what we can tell on recent project releases.